src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Microsoft.DotNet.Build.Tasks.Feed.targets

```diff
@@ -33,4 +38,18 @@
                       PublishFlatContainer="$(PublishFlatContainer)"
                       Overwrite="$(Overwrite)" />
     </Target>
-</Project>
```

Comment:
I expect that our core repos will likely just import this targets file and call the PushToBlobFeed directly.